doc/userguide/configuration/exception-policies.rst

```diff
@@ -0,0 +1,71 @@
+.. _exception policies:
+
+Exception Policies
+==================
+
+Suricata has a set of configuration variables to indicate what should the engine
+do when certain exception conditions, such as hitting a memcap, are reached.
+There are two categories for such settings:
+
+- exception policies - configurable via suricata.yaml; called when Suricata
+  reaches exception states
+- simulation options - exposed in debug mode and passed via command-line; used
+  to force failures or errors
+
+Exception Policies
+------------------
+
+Exception policies are implemented for:
+
+- stream (``stream.memcap``): If stream session or reassembly memcaps are hit, call
+  the memcap policy on the packet and flow.
+- stream (``stream.midstream``): If a session is picked up midstream, call the
+  memcap policy on the packet and flow.
+- flow (``flow.memcap``): Apply policy when memcap is reached and no flow could be
+  freed up.
+- defrag (``defrag.memcap``): Apply policy when no tracker could be picked up.
+- app-layer (``app-layer.error-policy``): Apply policy if a parser reaches an error state.
+
+To change any of those, check the corresponding section in the suricata.yaml
+file. The possible values, and their resulting behaviors, are:
+
+- ``drop-flow``: disable inspection for the whole flow (packets, payload,
+  application layer protocol), and drop the flow.
+- ``drop-packet``: disable payload and packet inspection, and drop the packet.
```

Comment:
lead with "drop the packet", then also something like no more inspection on it either (which seems implied by dropping, so not sure if it adds things)